app/templates/share_url.html.twig

```diff
@@ -0,0 +1,4 @@
+<div class="input-field">
+  <input value="{{url}}" id="game_url" type="text" class="validate">
```

Comment:
Why use an input field ?

Reply:
Because of the pretty subtitle, but you right it's not the aim an input field.